modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/data/set/provider/PendingCommerceOrderItemFDSActionProvider.java

```diff
@@ -98,7 +98,8 @@ public List<DropdownItem> getDropdownItems(
 				return _modelResourcePermission.contains(
 					PermissionThreadLocal.getPermissionChecker(), commerceOrder,
 					ActionKeys.UPDATE) &&
-					   (count > 0);
+					   ((count > 0) ||
+						cpInstance.hasCPInstanceUnitOfMeasures());
```

Comment:
Can we check the UOM in the orderItem instead of the cpInstance? In this way we don't need the new method in the CPInstanceImpl.java

Reply:
I think one can make the edit only if the SKU has more then one UOM, the only way to check it is through the CPInstance check because in the OrderItem we save only the Key of the UOM selected

Reply:
But right now also `cpInstance.hasCPInstanceUnitOfMeasures()` returns true if > 0 (so also with one single UOM)